subcommittees/policy-CoC/README.md

```diff
@@ -0,0 +1,33 @@
+# The Code of Conduct Policy Subcommittee
+
+## Purpose and Scope
+
+Enforce the Code of Conduct (CoC) in all parts of the Carpentries community
```

Comment:
Link to the canonical source(s) for the Code of Conduct and related documents?  Those are written under the guidance of this subcommittee, so it should be easy for folks reading about this subcommittee to find them.  This will also be helpful for future subcommittee members (e.g. so they know where to request updates if the subcommittee's recommendations change).